lib/libutils/isoc/bget_malloc.c

```diff
@@ -312,6 +312,7 @@ static void *raw_malloc(size_t hdr_size, size_t ftr_size, size_t pl_size,
 			struct bpoolset *poolset)
 {
 	void *ptr = NULL;
+	bufsize bs;
 	size_t s;
```

Comment:
Wouldn't it be enough to change the type of `s` to `bufsize` instead?

Reply:
Indeed. I thought the `_OVERFLOW` macros had to operate on arguments of the same type, but apparently not! Will fix.

Reply:
Yeah, it took some extra effort to support that. :-)